src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -158,81 +127,42 @@ public Random rng() {
         return (this.rng);
     }
 
-    /**
-     *
-     * @return
-     */
-    public URL getDatabaseDDL() {
-        return (this.getDatabaseDDL(this.workConf.getDBType()));
-    }
-
     /**
      * Return the URL handle to the DDL used to load the benchmark's database
      * schema.
-     * @param conn
+     *
+     * @param db_type
      * @throws SQLException
      */
-    public URL getDatabaseDDL(DatabaseType db_type) {
-        String ddlNames[] = {
-            this.benchmarkName + "-" + (db_type != null ? db_type.name().toLowerCase() : "") + "-ddl.sql",
-            this.benchmarkName + "-ddl.sql",
-        };
-
-        for (String ddlName : ddlNames) {
-            if (ddlName == null) continue;
-            URL ddlURL = this.getClass().getResource(DDLS_DIR + File.separator + ddlName);
-            if (ddlURL != null) {
-                if (LOG.isDebugEnabled())
-                    LOG.debug("Found DDL file for " + db_type + ": " + ddlURL );
-                return ddlURL;
-            }
-        } // FOR
-        LOG.trace(ddlNames[0]+" :or: "+ddlNames[1]);
-        LOG.error("Failed to find DDL file for " + this.benchmarkName);
-        return null;
-    }
+    public String getDatabaseDDLPath(DatabaseType db_type) {
```

Comment:
This changes the default ddl resolution logic, but the new way is pretty good too